src/main/kotlin/edu/cornell/cs/apl/viaduct/selection/ProtocolSelector.kt

```diff
@@ -0,0 +1,11 @@
+package edu.cornell.cs.apl.viaduct.selection
```

Comment:
This module could use a union function that combines the output of multiple selectors. Something like

```Kotlin
fun unions(vararg selectors: ProtocolSelector): ProtocolSelector
```

This would be implemented by a private class that internally maintains a list of selectors and calls them in order.